web/themes/custom/votegov/src/sass/components/hero.scss

```diff
@@ -57,11 +58,27 @@
   @include at-media-max('tablet-lg') {
     @include grid-container;
   }
+
+  .usa-button {
+    font-family: $theme-font-type-sans;
+    font-size: 16x;
+    width: auto;
+
+    @include at-media-max('mobile') {
+      width: 193px;
+    }
+  }
```

Comment:
I don't understand why any of these styles are necessary can you describe why they are needed. I'm not seeing the affects.

Reply:
You're right I don't think this is necessary since the buttons are styled globally